tests/RedshiftDatatypeTest.php

```diff
@@ -24,113 +24,33 @@ public function testInvalidType()
         }
     }
 
-    public function testValidNumericLengths()
-    {
-        new Redshift("numeric");
```

Comment:
Nemůžeš to smazat, pokud to nepřevedeš na ten nový dataprovider.

Reply:
v novém chybí test na velké/malé v typu

Reply:
chybí INT

Reply:
jj INT chybí, protože se pro int nevaliduje délka
https://github.com/keboola/php-datatypes/blob/master/src/Definition/Redshift.php#L149
to bych udělal v dalším PR

Reply:
To že se to nevaliduje s tím přece vůbec nesouvisí, ne? Co když je to rozbité když tam pošleš int nebo INT? Co když to vyhodí exception? Ten test zachycuje aktuální chování. Až to začneš validovat, tak tam přidáš další testy na validaci. Ale tady by měl být test na to, že to jde stejně jako to šlo předtím, než jsi ten test upravil..

Reply:
ok, přidal jsem to tam